spec/unit/proxy_spec.rb

```diff
@@ -1,15 +1,13 @@
-require "spec_helper"
-
 module Spank
   describe Proxy do
-    let(:sut) { Proxy.new(target) }
+    subject { Proxy.new(target) }
     let(:target) { double("target", :greet => nil) }
 
     context "when invoking a method" do
-      before { sut.greet('blah') }
+      before { subject.greet('blah') }
 
-      it "should send the message to the target" do
-        target.should have_received(:greet).with('blah')
+      it "sends the message to the target" do
+        expect(target).to have_received(:greet).with('blah')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.